### DISCUSSION TITLE
r/my-rv-is-registered

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#30] [Announcements]
# r/my-rv-is-registered

血泪，房车终于上牌了

https://blog.ops-coffee.cn/r/my-rv-is-registered

**Comment by halolake** [comment]
你这也太难了！我是在杭州上牌的，没找黄牛，自己跑了检测站和车管所，一共花了不到2天时间，还算顺利。

**Reply by ops-coffee** [reply]
特别难！

**Comment by xiaozhaoqi** [comment]
这也太麻烦了，现在2025会不会好一些了，直接买大厂家的房车？
